webpack.dev.js

```diff
@@ -1,40 +1,16 @@
 
 const webpack = require('webpack');
+const merge = require('webpack-merge');
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const CleanWebpackPlugin = require('clean-webpack-plugin');
 const HtmlWebpackTemplate = require('html-webpack-template');
+const common = require('./webpack.common');
 
-
-module.exports = {
-  entry: ['./browser/src/react-redux/index.js'],
+module.exports = merge(common, {
   mode: 'development',
-  module: {
-    rules: [
-      {
-        test: /jsx?$/,
-        exclude: /node_modules/,
-        use: {
-          loader: 'babel-loader',
-        },
-      },
-      {
-        test: /\.scss$/,
-        use: [
-          'style-loader',
-          'css-loader',
-          'sass-loader',
-        ],
-      },
-    ],
-  },
   resolve: {
     extensions: ['.js'],
```

Comment:
Is there a reason we only resolve for `.js` here but test for `/jsx?$/` in the common file?